website/blog/2024-11-07-bedrock-chat-model-part-1/index.md

```diff
@@ -0,0 +1,1235 @@
+---
+title: Using Bedrock Agent as an MLflow ChatModel with Tracing
+description: A guide for using BedRock Runtime Agent with ChatModel and custom trace handling.
+slug: bedrock-chat-model-part-1
+authors: [jas-bali]
+tags: [genai, pyfunc, bedrock, tracing]
+thumbnail: /img/blog/bedrock-chatmodel.png
+---
+
+![Thumbnail](bedrock-chatmodel.png)
+
+In this blog post, we delve into the integration of AWS Bedrock Agent as a ChatModel within MLflow, focusing on how to
+leverage Bedrock's Action Groups and Knowledge Bases to build a conversational AI application. The blog will guide you
+through setting up the Bedrock Agent, configuring Action Groups to enable custom actions with Lambda, and utilizing knowledge bases
+for context-aware interactions. A special emphasis is placed on implementing tracing within MLflow.
+By the end of this article, you'll have a good understanding of how to combine AWS Bedrock's advanced features
+with MLflow's capabilities such as agent request tracing, model tracking and consistent signatures for input examples.
+
```

Comment:
It would be great to have a high-level diagram explaining what we're about to build here using excalidraw